start_esp/fetch_service_config.py

```diff
@@ -114,6 +119,60 @@ def fetch_access_token(metadata):
     token = json.loads(response.data)["access_token"]
     return token
 
+def fetch_latest_rollout(service_name, access_token):
+    """Fetch rollouts"""
+    if access_token is None:
+        headers = {}
+    else:
+        headers = {"Authorization": "Bearer {}".format(access_token)}
+
+    client = urllib3.PoolManager(ca_certs=certifi.where())
+
+    page_token = ""
```

Comment:
You can add filter to only get "status=success" and page_size=1 to fetch the last success one.